crates/validation/macros/src/lib.rs

```diff
@@ -108,6 +108,7 @@ pub fn validated_type(input: TokenStream) -> TokenStream {
                 #validated_type_ident,
                 ::validation::ValidationError
             > {
+                panic!("here");
```

Comment:
is this debugging code? :)

Reply:
Yeah 😞 There may be a few instances of this before the PR is ready. Most of the advice I found when searching how to debug macro issues was along the lines of "just add debug statements until you figure it out." 

If anyone has a better way to debug macros, please let me know.

Reply:
no, that's about how i do it.

no worries about giving up on some of the packages. we can do those one-by-one.